helm/oncall/templates/_env.tpl

```diff
@@ -517,15 +553,43 @@
 {{- end }}
 
 {{- define "snippet.redis.env" -}}
+- name: REDIS_PROTOCOL
+  value: {{ include "snippet.redis.protocol" . }}
 - name: REDIS_HOST
   value: {{ include "snippet.redis.host" . }}
 - name: REDIS_PORT
-  value: "6379"
+  value: {{ include "snippet.redis.port" . }}
+- name: REDIS_DATABASE
+  value: {{ include "snippet.redis.database" . }}
+{{- with .Values.externalRedis.username }}
+- name: REDIS_USERNAME
+  value: {{ . | quote }}
+{{- end }}
 - name: REDIS_PASSWORD
   valueFrom:
     secretKeyRef:
       name: {{ include "snippet.redis.password.secret.name" . }}
       key: {{ include "snippet.redis.password.secret.key" . | quote}}
+{{- if and (not .Values.postgresql.enabled) .Values.externalRedis.ssl_options.enabled }}
```

Comment:
referencing `.Values.postgresql.enabled` seems like a typo here

Reply:
It is not a typo. The option `.Values.postgresql.enabled` is used for deploying PostgreSQL as the chart dependency (new dedicated cluster in the same namespace). This cluster has no SSL options. The option `.Values.externalRedis` defines the external cluster, and SSL configuration is possible only for it. If someone set to use internal Postgres deployment (for development, of course) and set `.Values.externalRedis.ssl_options` then SSL options should not be picked to manifest.

Thus, this is the right expression.

Reply:
Yes, it is typo, thank you :)